apiserver/logsink.go

```diff
@@ -175,9 +218,10 @@ func (h *logSinkHandler) sendError(w io.Writer, req *http.Request, err error) {
 }
 
 // logToFile writes a single log message to the logsink log file.
-func (h *logSinkHandler) logToFile(prefix string, m params.LogRecord) error {
-	_, err := h.fileLogger.Write([]byte(strings.Join([]string{
+func logToFile(logger io.Writer, prefix string, m params.LogRecord) error {
```

Comment:
logger is often used as a global loggo.Logger variable name. Perhaps use "writer" here?